README.md

```diff
@@ -4,17 +4,17 @@
 
 Read in [Português (Brasileiro)](https://github.com/whyboris/Video-Hub-App/blob/master/README.br.md)
 
-## Download now
+## Download Now
 
 This software is available for $3.50 through [videohubapp.com](https://videohubapp.com/download.html)
 
-$3.50 of every sale goes to the [_cost-effective_](https://www.givewell.org/charities/top-charities) charity [Against Malaria Foundation](https://www.againstmalaria.com/).
+$3.50 of every sale goes to the [_cost-effective_](https://www.givewell.org/charities/top-charities) charity of [Against Malaria Foundation](https://www.againstmalaria.com/).
```

Comment:
I think the "of" is not needed grammatically 🤔 

Example: "I drove across the bridge Golden Gate" or "A part of my spending money goes to the musical band Pink Floyd"

Thoughts?